data/json/items/generic.json

```
@@ -2408,7 +2408,7 @@
     "name": "cash card",
     "description": "A yellow plastic card used to store money.  These became popular once the government officially moved to all electronic money.  It holds up to 2 million dollars.",
     "price": 100,
```

```suggestion
    "price": 100,
```
Should this also be changed to 1000?